python/lsst/ts/imsim/utils/utility.py

```diff
@@ -129,3 +129,40 @@ def getZkFromFile(zkFilePath):
         zk[key] = np.fromstring(val[0], sep=" ")
 
     return zk
+
+
```

Comment:
Is it possible to add a quick test here just to make sure it's working as expected?